MAUI/Calendar/getting-started.md

```diff
@@ -105,9 +105,9 @@ this.Calendar.View = CalendarView.Month;
 
 ## Change first day of week
 
-The Calendar control is rendered with `Sunday` as the first day of the week and it allows customization to change the first day of the week using the `FirstDayOfWeek` property in month view.
+The Calendar control rendered with `Sunday` as first day of week and it  provides customization to change the first day of the week with the FirstDayOfWeek property in month view.
```

Comment:
The Calendar control is rendered with `Sunday` as the first day of the week and it allows customization to change the first day of the week using the `FirstDayOfWeek` property in month view.

Reply:
Content changed for first day of week